lib/sycamore/sycamore/functions/document.py

```diff
@@ -46,15 +48,17 @@ def split_and_convert_to_image(doc: Document) -> list[Document]:
     elements_by_page: dict[int, list[Element]] = {}
 
     for e in doc.elements:
-        page_number = e.properties["page_number"]
+        page_number = e.properties[DocumentPropertyTypes.PAGE_NUMBER]
         elements_by_page.setdefault(page_number, []).append(e)
 
     new_docs = []
     for page, image in enumerate(images):
         elements = elements_by_page.get(page + 1, [])
         new_doc = Document(binary_representation=image.tobytes(), elements=elements)
         new_doc.properties.update(doc.properties)
-        new_doc.properties.update({"size": list(image.size), "mode": image.mode, "page_number": page + 1})
+        new_doc.properties.update(
+            {"size": list(image.size), "mode": image.mode, DocumentPropertyTypes.PAGE_NUMBER: page + 1}
```

Comment:
Why are `size` and `mode` not named constants?

Reply:
I just updated page_number in a previous pr since it was used all over. Dunno where size and mode are currently used so didn't touch